sources/packages/backend/workflow/test/2025-2026/parttime-assessment/eligibility/parttime-assessment-eligibility-CSGD.e2e-spec.ts

```diff
@@ -25,6 +26,10 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-CSGD
           DependentEligibility.Eligible0To18YearsOld,
           { referenceDate: assessmentConsolidatedData.offeringStudyStartDate },
         ),
+        createFakeStudentDependentEligible(
```

Comment:
Can we have the E2Es planned to test the different conditions for the dependents?
- Scenario 1
  - 11 Years or under where the eligibility must be true.
- Scenario 2
  - 12 years over on declared on taxes where the eligibility must be true.
- Scenario 3
  - Only no eligible dependents are present.

Are these scanrios not supposed be based on the dependents created using the `createFakeStudentDependentEligibleForChildcareCost`?